spec/controllers/facility_statements_controller_spec.rb

```diff
@@ -133,19 +135,26 @@ def create_order_details
 
       it_should_deny_all [:staff, :senior_staff]
 
-      it "should create and send statements" do
-        expect(@order_detail1.reload.reviewed_at).to be < Time.zone.now
-        expect(@order_detail1.statement).to be_nil
-        expect(@order_detail1.price_policy).not_to be_nil
-        expect(@order_detail1.account.type).to eq(@account_type)
-        expect(@order_detail1.dispute_at).to be_nil
-
-        grant_and_sign_in(@user)
-        do_request
-        expect(flash[:error]).to be_nil
-        expect(assigns(:account_statements)).to have_key(@account)
-        expect(assigns(:account_statements)).to have_key(@account2)
-        expect(response).to redirect_to action: :new
+      context "when statement emailing is on", feature_setting: { send_statement_emails: true } do
+        include_examples "it sets up order_detail and creates statements"
+
+        it "sends statements" do
+          grant_and_sign_in(@user)
+          do_request
+
+          expect(ActionMailer::Base.deliveries).not_to be_empty
```

Comment:
I could be wrong, but I don't believe we do `ActionMailer::Base.deliveries.clear` always, so this it's possible that this test is inaccurate. For these kind of things, I like `expect { do_request }.not_to change(ActionMailer::Base.deliveries, :count)` and `expect { do_request }.to change(ActionMailer::Base.deliveries, :count).by(1)`